packages/token-lists/schema/akkafinance.json

```diff
@@ -0,0 +1,369 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "akkafinance",
+    "title": "AKKA Token List",
+    "description": "Schema for lists of tokens compatible with the AKKA Interface, including Uniswap standard and AKKA Aptos",
+    "definitions": {
```

Comment:
What are the differences between this custom schema and the pancakeswap/uniswap schema?

Reply:
It's because tokens have wierd names and pattern of pancake does not support